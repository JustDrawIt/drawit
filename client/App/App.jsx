import React from 'react';
import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
import ScreeenLogin from '../screens/ScreenLogin';
import ScreenCreateGame from '../screens/ScreenCreateGame';
import ScreenGame from '../screens/ScreenGame';

const App = () => (
  <BrowserRouter>
    <Switch>
      <Route path="/login" component={ScreeenLogin} />
      <Route path="/create_game" component={ScreenCreateGame} />
      <Route path="/game/:id" component={ScreenGame} />
      <Redirect to="/login" />
    </Switch>
  </BrowserRouter>
);

export default App;
