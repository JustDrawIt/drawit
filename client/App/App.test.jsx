import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('<App />', () => {
  test('App Exists', () => {
    const { baseElement } = render(<App />);
    expect(baseElement).toBeDefined();
  });
});
