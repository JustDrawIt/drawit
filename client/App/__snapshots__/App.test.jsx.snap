// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches snapshot 1`] = `
<div>
  <div
    class="css-1kwl4wt"
  >
    <div
      class="css-17ijup6"
    >
      <h1>
        Draw It!
      </h1>
      <div>
        <input
          class="css-112ujqv"
          placeholder="Join code"
          type="text"
        />
        <button
          class="css-hk4u7l"
        >
          Play!
        </button>
      </div>
      <span>
        or
      </span>
      <div
        class="css-hl79y"
      >
        <button
          class="css-9c6b8o"
          color="primary"
        >
          <a
            href="/create_game"
          >
            Create a game
          </a>
        </button>
      </div>
    </div>
  </div>
</div>
`;
